internal/service/rds/cluster_activity_stream_test.go

```diff
@@ -161,12 +168,8 @@ func testAccCheckAWSClusterActivityStreamDestroyWithProvider(s *terraform.State,
 	return nil
 }
 
-func testAccAWSClusterActivityStreamConfigBase(clusterName, instanceName string) string {
-	return fmt.Sprintf(`
-data "aws_availability_zones" "available" {
-  state = "available"
-}
-
+func testAccClusterActivityStreamConfigBase(clusterName, instanceName string) string {
+	return acctest.ConfigCompose(acctest.ConfigAvailableAZsNoOptIn(), fmt.Sprintf(`
```

Comment:
Otherwise:

```console
% make testacc TESTS=TestAccAWSRDSClusterActivityStream_ PKG=rds
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/rds/... -v -count 1 -parallel 20 -run='TestAccAWSRDSClusterActivityStream_'  -timeout 180m
=== RUN   TestAccAWSRDSClusterActivityStream_basic
=== PAUSE TestAccAWSRDSClusterActivityStream_basic
=== RUN   TestAccAWSRDSClusterActivityStream_disappears
=== PAUSE TestAccAWSRDSClusterActivityStream_disappears
=== CONT  TestAccAWSRDSClusterActivityStream_basic
=== CONT  TestAccAWSRDSClusterActivityStream_disappears
=== CONT  TestAccAWSRDSClusterActivityStream_basic
    cluster_activity_stream_test.go:29: Step 1/2 error: Error running apply: exit status 1
        
        Error: error creating RDS cluster: InvalidVPCNetworkStateFault: Availability zones '[us-west-2-lax-1b, us-west-2-lax-1a, us-west-2-wl1-den-wlz-1]' are unavailable in this region, please choose another zone set.
        	status code: 400, request id: 5d4648ea-75e3-4822-b081-8d89923fa541
        
          with aws_rds_cluster.test,
          on terraform_plugin_test.tf line 11, in resource "aws_rds_cluster" "test":
          11: resource "aws_rds_cluster" "test" {
        
=== CONT  TestAccAWSRDSClusterActivityStream_disappears
    cluster_activity_stream_test.go:66: Step 1/1 error: Error running apply: exit status 1
        
        Error: error creating RDS cluster: InvalidVPCNetworkStateFault: Availability zones '[us-west-2-lax-1b, us-west-2-lax-1a, us-west-2-wl1-den-wlz-1]' are unavailable in this region, please choose another zone set.
        	status code: 400, request id: db3c741f-34b1-41b9-9459-bf6589694d2e
        
          with aws_rds_cluster.test,
          on terraform_plugin_test.tf line 11, in resource "aws_rds_cluster" "test":
          11: resource "aws_rds_cluster" "test" {
        
--- FAIL: TestAccAWSRDSClusterActivityStream_basic (11.37s)
--- FAIL: TestAccAWSRDSClusterActivityStream_disappears (11.43s)
FAIL
FAIL	github.com/hashicorp/terraform-provider-aws/internal/service/rds	15.237s
FAIL
make: *** [testacc] Error 1
```